docs/user_workloads/accessing_virtual_machines.md

```diff
@@ -161,21 +161,24 @@ This allows for dynamic injection of SSH public keys at runtime by updating the
 
 Please note that new Secrets cannot be attached to a running VM: You must restart the VM to attach the new Secret.
 
-> Note: This requires the qemu-guest-agent to be installed within the guest.
->
-> Note: When using qemuGuestAgent propagation,
-> the `/home/$USER/.ssh/authorized_keys` file will be owned by the guest agent.
+> **Requirement:** The qemu-guest-agent must be installed within the guest.
+
+> **Deprecation Notice:** The implementation supporting qemu-guest-agent versions older than 5.2 is deprecated
+> and will stop working in a future release.
+
+> **Important:** When using `qemuGuestAgent` propagation,
```

Comment:
```suggestion
!!! Important 
    When using `qemuGuestAgent` propagation method,
```
You'd need to replace the subsequent '>' characters and add the 4 spaces (as here) to keep it in the same admonition.
I think this also belongs above the 'requirements' heading in the opening text.